packages/http-client/src/remote-admin-api.ts

```diff
@@ -7,40 +7,57 @@ import { StreamID } from '@ceramicnetwork/streamid'
 export class RemoteAdminApi implements AdminApi {
   // Stored as a member to make it easier to inject a mock in unit tests
   private readonly _fetchJson: typeof fetchJson = fetchJson
+
   readonly baseURL = './admin/models'
 
   constructor(
-    private readonly _apiUrl: URL
+    private readonly _apiUrl: URL,
+    private readonly _context: Context
   ) {}
 
   private getUrl(): URL {
     return new URL(this.baseURL, this._apiUrl)
   }
 
+  private async buildAuthorizationHeader(modelsIDs?: Array<StreamID>): Promise<string> {
+    const body = modelsIDs ? modelsIDs.map(streamID => streamID.toString()) : undefined
+    const jws = await this._context.did.createJWS({
+      timestamp: Date.now(),
```

Comment:
what's this timestamp for?  Warning bells always go off for me when I see timestamps being taken from the system clock, especially the system clock on the client machine, as we cannot trust that timestamp for just about anything

Reply:
To prevent replay attacks. If someone gets ahold of your signature after the fact they could resend it to the node to reissue the same command. 
What we really should do here is to have a challenge response protocol, but this would increase the scope.

Reply:
Actually might be worth it. We would need to do the following:

1. Make all admin endpoints not require any JWS
2. Make all endpoints return a random challenge when requested
3. Add a special endpoint to respond to the challenge

Note that this would add additional complexity to the implementation because it needs to be stateful.

Reply:
yeah, this does kind of seem worth doing IMO, much better than relying on a timestamp.  Could probably wait until after devcon though?

Reply:
@stbrody Wdym? Either we do it now or we wait with adding admin api before devcon. Otherwise there will be a breaking change here.

Reply:
The simplest way would probably be to do this:

1. Add a new endpoint `/admin/getCode`. This returns a random string / number, stores it in memory with a timestamp
2. For all other `/admin/` endpoints remove the timestamp argument and add a new `code` argument. 
3. When authorizing a request make sure `code` is valid and was issued within T time (e.g. 1 min), then remove the code from memory

cc @pawartur

Reply:
FWIW I don't think it'd be the worst thing in the world to have a breaking change here, if we wanted to get this out the door for devcon.  Especially since it's only to the admin API, no live apps would break, only their deployment process would, and they'd just need to upgrade their composedb cli to the newest version and then it would work again.